### DISCUSSION TITLE
Connecting people

### DISCUSSION THREAD

**Discussion by orabeidm** [ps-iowned/ps-iowned#3] [General]
https://4pda.to/forum/index.php?showtopic=1053197

"Write me a private message on this forum and I'll give you the contacts of a like-minded person who is also working on the same thing and has achieved some success."

**Comment by the-connors** [comment]
> Ваш уровень предупреждений был повышен.
>- - - - - - - - - - - - -
>Спамер.
>- - - - - - - - - - - - -
>Спамерам не место на 4PDA. Аккаунт заблокирован.
>
>Ваш аккаунт заблокирован. Вам доступен только раздел "[Последний шанс](https://4pda.to/forum/index.php?showforum=22)".
>- - - - - - - - - - - - -
>Напоминаем, что любое нарушение Правил ресурса повышает ваш Уровень Предупреждений (УП) на 20%. При достижении 100% произойдет автоматическое блокирование вашего аккаунта и дальнейшее пребывание на 4pda будет невозможным.
>
>Процедура снижения Уровня Предупреждений прописана в [Правилах ресурса](https://4pda.to/forum/index.php?act=boardrules). Рекомендуем вам с ними ознакомиться.
>- - - - - - - - - - - - -
>
>Мы хорошо понимаем, что вы можете быть расстроены возникшей ситуацией, но у нас есть Правила, которые при регистрации Вы согласились соблюдать.
>
>Прежде чем написать представителю Администрации в ответ на нарушение - рекомендуем успокоиться и хорошо подумать.
>
>Обращаем ваше внимание, что в случае хамства в адрес любого представителя Администрации, аккаунт нарушителя блокируется незамедлительно, а дальнейшее пребывание на 4pda становится невозможным.
>
>Искренне надеемся на ваше понимание.

**Reply by the-connors** [reply]
Other team wants no collaboration...

> Ваш уровень предупреждений был повышен.
> - - - - - - - - - - - - -
>
> Контрольный
> - - - - - - - - - - - - -
> Ваш аккаунт заблокирован.
>
> Ваш аккаунт полностью заблокирован.

**Comment by orabeidm** [comment]
Write your Telegram nickname.

**Comment by orabeidm** [comment]
send it to me

orabeidm@gmail.com

**Comment by the-connors** [comment]
> Write your Telegram nickname.

No phone. Sorry. Telegram needs phone.

**Reply by orabeidm** [reply]
everyone has a phone. if you are afraid to burn the phone number, just hide it in the telegram.

**Comment by the-connors** [comment]
> send it to me
> 
> [orabeidm@gmail.com](mailto:orabeidm@gmail.com)

Sent email.

**Comment by orabeidm** [comment]
In any case, my guys need the code or dump of the microcontroller and bootloader. If you have it, send it to me not by email. I will pass it on to whoever needs it.
